FirebaseDynamicLinks/README.md

```diff
@@ -0,0 +1,24 @@
+# Firebase Dynamic Links SDK for iOS
```

Comment:
Nit: can you please format this to ensure the lines don't go over 100 characters? For links you can see how we do it in the regular README:

https://raw.githubusercontent.com/firebase/firebase-ios-sdk/master/README.md

Reply:
I did the formating where ever possible. please see